contracts/contracts/ProjectRegistry.sol

```diff
@@ -16,24 +17,6 @@ contract ProjectRegistry {
         MetaPtr metadata;
     }
 
-    // A linked list of owners of a project
-    // The use of a linked list allows us to easily add and remove owners,
-    // access them directly in O(1), and loop through them.
-    //
-    // {
-    //     count: 3,
-    //     list: {
-    //         OWNERS_LIST_SENTINEL => owner1Address,
-    //         owner1Address => owner2Address,
-    //         owner2Address => owner3Address,
-    //         owner3Address => OWNERS_LIST_SENTINEL
-    //     }
-    // }
-    struct OwnersList {
```

Comment:
@schultztimothy I wouldn't put this struct in another file since it's used only here. 
maybe we could move the MetaPtr cause it's shared everywhere in our projects, wdyt?

Reply:
especially becuase we have its doc and all the methods to manage it inside the contract

Reply:
Sounds good to me. I thought it might be able to be used across projects in a similar way, but we can always move it out later